invenio_communities/serializers/schemas/community.py

```diff
@@ -38,6 +40,9 @@ class CommunitySchemaV1(Schema):
     page = fields.String()
     curation_policy = fields.String()
     last_record_accepted = fields.DateTime()
+    link = fields.Function(
```

Comment:
Did you check the behaviour on Zenodo? I'm worried that request.url_root is actually ``http://localhost:500/api/`` when you're in the API app - hence this also won't work